cucumber/api/features/change_password.feature

```diff
@@ -10,14 +10,43 @@ Feature: Change the password of a role
 
   Scenario: With basic authentication, users can update their own password using the current password.
 
-    Given I set the password for "alice" to "my-password"
-    When I successfully PUT "/authn/cucumber/password" with username "alice" and password "my-password" and plain text body "new-password"
-    Then I can GET "/authn/cucumber/login" with username "alice" and password "new-password"
+    Given I set the password for "alice" to "My-Password1"
+    When I successfully PUT "/authn/cucumber/password" with username "alice" and password "My-Password1" and plain text body "New-Password1"
+    Then I can GET "/authn/cucumber/login" with username "alice" and password "New-Password1"
 
   Scenario: With basic authentication, users can update their own password using the current API key.
 
-    When I successfully PUT "/authn/cucumber/password" with username "alice" and password ":alice_api_key" and plain text body "new-password"
-    Then I can GET "/authn/cucumber/login" with username "alice" and password "new-password"
+    When I successfully PUT "/authn/cucumber/password" with username "alice" and password ":alice_api_key" and plain text body "New-Password1"
+    Then I can GET "/authn/cucumber/login" with username "alice" and password "New-Password1"
+
+  Scenario: Succeed to set password applying to password complexity.
```

Comment:
I believe this should be split into 3 scenarios

Reply:
Remove tests covered in UT and left only one in integration.